AWS/cloudformation-templates/heroku-app-and-cloudfront.json

```diff
@@ -145,7 +145,37 @@
           }
         }
       }
+    },
+    "DeployUser": {
+      "Type": "AWS::IAM::User",
+      "Condition": "CreateDeployAccount",
```

Comment:
You'll need to copy-paste the condition statement in this template as well as the parameter controlling that condition statement